ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -0,0 +1,148 @@
+                    // before normalization.
+                    // If data shape is [N,C,H,W], the function returns
+                    // [N, num_groups, C // num_groups, H, W]
+                    std::shared_ptr<ngraph::Node>
+                        create_group_norm_shape(const Output<ngraph::Node>& data, size_t num_groups)
+                    {
+                        const auto& pshape = data.get_partial_shape();
+                        NGRAPH_CHECK(pshape.rank().is_static());
+                        size_t rank_size = pshape.rank().get_length();
+                        NGRAPH_CHECK(rank_size >= 3, "3-D and above tensors supported only");
+
+                        if (pshape.is_static())
+                        {
+                            const auto& shape = pshape.to_shape();
+                            std::vector<size_t> new_shape{
+                                shape[0], num_groups, shape[1] / num_groups};
+                            for (size_t i = 2; i < rank_size; i++)
+                            {
+                                new_shape.push_back(shape[i]);
+                            }
+                            return default_opset::Constant::create(
+                                element::i64, Shape{new_shape.size()}, new_shape);
+                        }
+
+                        auto shape = std::make_shared<default_opset::ShapeOf>(data);
+                        auto splits = builder::opset1::split(shape, rank_size);
+                        auto num_groups_const =
+                            default_opset::Constant::create(element::i64, Shape{1}, {num_groups});
+                        NodeVector new_shape{
+                            splits[0].get_node_shared_ptr(),
+                            num_groups_const,
+                            std::make_shared<default_opset::Divide>(splits[1], num_groups_const)};
+                        for (size_t i = 2; i < rank_size; i++)
+                        {
+                            new_shape.push_back(splits[i].get_node_shared_ptr());
+                        }
+                        return std::make_shared<default_opset::Concat>(new_shape, 0);
+                    }
+                }
+            } // detail
+
+            namespace set_1
+            {
+                OutputVector group_norm(const Node& node)
+                {
+                    auto inputs = node.get_ng_inputs();
+                    NGRAPH_CHECK(inputs.size() == 3,
+                                 "Invalid number of inputs. Expected 3, actual " +
+                                     std::to_string(inputs.size()));
+
+                    auto data = inputs[0];
+                    auto scale = inputs[1];
+                    auto bias = inputs[2];
+
+                    size_t num_groups =
+                        static_cast<size_t>(node.get_attribute_value<int64_t>("num_groups"));
+                    float eps = node.get_attribute_value<float>("eps", 1e-5);
```

Comment:
In the code of the MO extractor for this op (https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/extensions/front/onnx/group_norm_ext.py#L45), the default value for eps is 1e-6. Why is it 1e-5 here?

Reply:
I was suggested by the paper https://arxiv.org/pdf/1803.08494.pdf and pytorch definition https://pytorch.org/docs/stable/generated/torch.nn.GroupNorm.html#torch.nn.GroupNorm - we have 1e-5 there. 
I can adjust those two. Which value should I pick?

Reply:
I don't know why MO uses that value. @lazarevevgeny probably knows. At least we have to use the same default value in all components.